VERSIONING.md

```diff
@@ -0,0 +1,202 @@
+# Cosign Versioning Policy
+
+This is the versioning policy regarding the stability of Cosign, both the API and CLI.
+
+**tl;dr (CLI):** semver for specific behaviors, but with deprecations.
+
+**tl;dr (API):** no guarantees at all (for now).
+
+Principles:
+
+- Provide well-versioned, very reliable tools/libraries that people can write
+  scripts/code against.
+- Provide places for experimentation and rapid-development, not covered under
+  versioning guarantees.
+- Be explicit about what is and isn’t covered.
+- Better to promise a weaker guarantee and stick to it consistently than to be
```

Comment:
`Better to promise a weaker guarantee and stick to it consistently`

? :p

Reply:
> ? :p

? :p

Not sure what the source of confusion is here. I meant to say something along the lines of "before, we had a stronger versioning guarantee but didn't actually follow it; I'd rather have a policy we intend to follow."

Is there a better way to word that?